music_assistant/controllers/media/radio.py

```diff
@@ -52,10 +56,11 @@ async def versions(
             all_versions.pop(prov_version.item_id, None)
 
         # return the aggregated result
-        return all_versions.values()
+        return list(all_versions.values())
```

Comment:
I am sure there is a reason that dict.values() doesn't return a list and this had me digging around some but from what I see, this is the preferred solution. Not a request for a change, just an observation.